dts/bindings/radio_fem/skyworks,sky66407-11.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+description: |
+    This is a representation of the Skyworks SKY66407-11 Radio Front-End module.
+    It is supplementing the generic representation of FEM with settling time and
+    gain values appropriate for SKY66407-11 device.
+
+compatible: "skyworks,sky66407-11"
+
+include: generic-fem-two-ctrl-pins.yaml
+
+properties:
+    ctx-settle-time-us:
+      type: int
+        default: 23
+        description: |
+            Settling time in microseconds from activation of CTX to transmit.
```

Comment:
I think you should use 2-space indentation to please the compliance check. So this would be:
```
properties:
  ctx-settle-time-us:
    type: int
    default: 23
    description: |
      Settling time in microseconds from activation of CTX to transmit.
```

Reply:
Ok Thanks. Hopefully I got it now . Thanks for your patience.